pkg/endpoint/bpf.go

```diff
@@ -211,7 +211,7 @@ func (e *Endpoint) addNewRedirectsFromDesiredPolicy(ingress bool, desiredRedirec
 				var finalizeFunc revert.FinalizeFunc
 				var revertFunc revert.RevertFunc
 
-				proxyID, err := e.ProxyID(e.desiredPolicy.NamedPortsMap, l4)
+				proxyID, err := e.ProxyID(l4)
```

Comment:
Typo is commit msg:

"of demand" -> "on demand"

Reply:
Fixed, thanks!